pom.xml

```diff
@@ -0,0 +1,65 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.camunda.connector</groupId>
+    <artifactId>connector-parent</artifactId>
+    <version>0.4.0-SNAPSHOT</version>
+    <relativePath/>
+  </parent>
+
+  <artifactId>connectors-bundle-parent</artifactId>
+  <version>${revision}</version>
+  <packaging>pom</packaging>
+
+  <name>connectors-bundle</name>
+  <description>Parent POM for out-of-the-box Connectors</description>
+  <inceptionYear>2022</inceptionYear>
+
+  <modules>
+    <module>connector</module>
+    <module>docker/bundle/mvn</module>
+  </modules>
+
+  <properties>
+    <revision>0.14.0-SNAPSHOT</revision>
```

Comment:
Could you detail why we are doing this via a property?

Reply:
The version plugin still works, we can use its `update-property` goal to set the revision. 

Maybe I'm wrong about the version plugin, my assumption was that we need to run it multiple times to achieve the same result as with this `revision` approach:

1. Bump the root pom version (`set` goal)
2. Bump the parent versions in each of the child poms (`update-parent` goal)

If that's the case, we'll need to adjust our build whenever we add a new module. With `revision`, on the other hand, we only need to manipulate the version in one place.

Please correct me if I'm wrong and we can do it in a single goal for the whole project :)

Reply:
Usually, for multi-module projects, you simply run `mvn versions:set -DnewVersion=0.4.0` and Maven adjusts the parent and all child modules directly in one go. This is what we do in the SDK as well.

Reply:
Oh ok. Then I agree that we don't actually need the revision approach.

Reply:
Saves us to add the flatten plugin unnecessarily. We simply don't have a use case for the CI-friendly approach here, I believe. But I might also not understand the benefits entirely yet.